tests/torchinfo_test.py

```diff
@@ -162,6 +163,18 @@ def test_resnet152() -> None:
     summary(model, (1, 3, 224, 224), depth=3)
 
 
+def test_pruning() -> None:
+    model = SingleInputNet()
+    for module in model.modules():
+        if isinstance(module, (torch.nn.Conv2d, torch.nn.Linear)):
+            # mypy: allow-untyped-calls
```

Comment:
Remove this line, needs to be `# type ignore[allow-untyped-calls]` on the next line